deepchem/models/torch_models/layers.py

```diff
@@ -3056,3 +3056,87 @@ def forward(self, inputs: torch.Tensor):
         atom_enbeddings = torch.nn.functional.embedding(atom_number,
                                                         self.embedding_list)
         return atom_enbeddings
+
+class Highway(nn.Module):
+    """ Create a highway layer. y = H(x) * T(x) + x * (1 - T(x))
+
+    H(x) = activation_fn(matmul(W_H, x) + b_H) is the non-linear transformed output
+    T(x) = sigmoid(matmul(W_T, x) + b_T) is the transform gate
+
+    Implementation based on paper
+
+    Srivastava, Rupesh Kumar, Klaus Greff, and Jürgen Schmidhuber. "Highway networks." arXiv preprint arXiv:1505.00387 (2015).
+
+
+    This layer expects its input to be a two dimensional tensor
+    of shape (batch size, # input features).  Outputs will be in
+    the same shape.
+    """
+
+    def __init__(self,
+                 activation_fn='ReLU',
+                 biases_initializer='zeros',
+                 weights_initializer=None,
+                 **kwargs):
+        """
+        Parameters
+        ----------
+        activation_fn: object
```

Comment:
object is too loose a type. Check the other layers to see what they do for activation functions